docs/root/configuration/http/http_filters/lua_filter.rst

```diff
@@ -143,8 +143,8 @@ The ``GLOBAL`` Lua script will be overridden by the referenced script:
   <envoy_v3_api_field_extensions.filters.http.lua.v3.Lua.inline_code>`. Therefore, do not use
   ``GLOBAL`` as name for other Lua scripts.
 
-Or We can define new Lua script in the LuaPerRoute configuration directly to override `GLOBAL`  
-Lua script as follow example configuration:
+Or we can define a new Lua script in the LuaPerRoute configuration directly to override the `GLOBAL`  
+Lua script as follows:
```

Comment:
While you're here, could you update the "Go Go C Plus Plus." with something else?

Reply:
Will do

Reply:
Done!